recipes-wigwag/maestro/maestro_0.0.1.inc

```diff
@@ -54,7 +54,7 @@ file://maestro-watcher.path \
 "
 
 SRCREV_FORMAT="m"
-SRCREV_m="d6ab0da4238bb4d8b767db3f70af71db909fae1c"
+SRCREV_m="458a91f7278e8431ab446c3b751500e2b390c262"
```

Comment:
What do these SHAs point to?

Since https://github.com/PelionIoT/maestro/pull/132 is still unmerged, won't this above SHA change once it's merged?

Reply:
Yes, we decided to lock the SHA to the latest release of the component. That SHA is pointing to the master branch and v2.7.0 release of maestro - https://github.com/PelionIoT/maestro/releases/tag/v2.7.0